scripts/asciidoc2html.py

```diff
@@ -32,7 +31,7 @@
 import io
 import pathlib
 
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.insert(0, str(pathlib.Path(__file__).parents[1]))
```

Comment:
For some reason, `__file__` was `scripts/dev/../asciidoc2html.py` here, causing `.parents[1]` to end up with `scripts/dev/` instead of `scripts/`. I added a `.resolve()` to fix it: eed0373feb1a1750aa834b9c8d3524ddb1833780